__tests__/app.test.js

```diff
@@ -25,20 +25,51 @@ describe("GET /api/topics", () => {
       .get("/api/topics/")
       .expect(200)
       .then(({ body: { topics } }) => {
-        expect(topics).toEqual([
-          {
-            description: "The man, the Mitch, the legend",
-            slug: "mitch",
-          },
-          {
-            description: "Not dogs",
-            slug: "cats",
-          },
-          {
-            description: "what books are made of",
-            slug: "paper",
-          },
-        ]);
+        expect(topics).toHaveLength(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: Takes a specific article_id and returns only the relevant article", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
```

Comment:
You're requesting article 4. This id at the very least should be 4! As you own the database you can, and should assert the values on article 4.